c_src/membrane_h264_ffmpeg_plugin/parser.c

```diff
@@ -83,6 +92,20 @@ UNIFEX_TERM parse(UnifexEnv *env, UnifexPayload *payload, State *state) {
     data_left -= ret;
 
     if (pkt->size > 0) {
+      if (resolution_changed(*last_res, state)) {
+        if (changes_cnt >= max_changes) {
+          max_changes *= 2;
+          changes = unifex_realloc(changes, max_changes * sizeof(resolution));
+        }
+
+        resolution new_res = {state->parser_ctx->width,
+                              state->parser_ctx->height, frames_cnt};
+        changes[changes_cnt] = new_res;
+        changes_cnt++;
```

Comment:
I think it is would be more common to write it as: `changes[changes_cnt++] = new_res`. Not a problem of course, just a note